app/app/components/OurTeam.jsx

```diff
@@ -0,0 +1,102 @@
+"use client"; // Mark this component as a client component
+
+import React, { useEffect, useRef, useState } from "react";
+import Image from "next/image"; // Import Image from Next.js
+import styles from "./OurTeam.module.css"; // Import the CSS module
+
+const OurTeam = () => {
+    const [members, setMembers] = useState([]); // State to hold the member data
+    const [visibleMembers, setVisibleMembers] = useState([]); // State for visibility
+    const membersRef = useRef([]);
+
+    useEffect(() => {
+        // Fetch members from JSON file
+        const fetchMembers = async () => {
+            const response = await fetch("/q-code/qcode-members.json"); // Adjust path if necessary
```

Comment:
This is a valid comment, cause it adds info for the reader, that this path might need to be changed